loop_rate:      10.0 # The frequency in Hz at which to run the control loop.
queue_size:     1    # The size of the message queue used for publishing and subscribing to topics.
